src/constants.ts

```diff
@@ -121,7 +121,8 @@ export enum EmbeddingModelProviders {
   OPENAI = "openai",
   COHEREAI = "cohereai",
   GOOGLE = "google",
-  AZURE_OPENAI = "azure_openai",
+  // AZURE_OPENAI = "azure_openai",
+  AZURE_OPENAI = "azure openai",
```

Comment:
does it belong to this PR?

Reply:
yeah,  it's a separate commit and @logancyang also learn about this.